proxy/tests/telemetry.rs

```diff
@@ -1088,3 +1091,70 @@ mod transport {
         assert_contains!(metrics.get("/metrics"), &expected);
     }
 }
+
+// https://github.com/runconduit/conduit/issues/613
+#[test]
+#[cfg_attr(not(feature = "flaky_tests"), ignore)]
+fn metrics_compression() {
+    let _ = env_logger::try_init();
+
+    let Fixture { client, metrics, proxy: _proxy } = Fixture::inbound();
+
+    let do_scrape = |encoding: &str| {
+        let resp = metrics.request(
+            metrics.request_builder("/metrics")
+                .method("GET")
+                .header("Accept-Encoding", encoding)
+        );
+
+        {
+            // create a new scope so we can release our borrow on `resp` before
+            // getting the body
+            let content_encoding = resp.headers()
+                .get("content-encoding")
+                .as_ref()
+                .map(|val| val
+                    .to_str()
+                    .expect("content-encoding value should be ascii")
+                );
+            assert_eq!(content_encoding, Some("gzip"),
+                "unexpected Content-Encoding {:?} (requested Accept-Encoding: {})", content_encoding, encoding);
+        }
+
+        let body = resp.into_body()
+            .concat2()
+            .wait()
+            .expect("response body concat");
+        let mut decoder = flate2::read::GzDecoder::new(body.into_buf());
+        let mut scrape = String::new();
+        decoder.read_to_string(&mut scrape)
+            .expect(&format!(
+                "decode gzip (requested Accept-Encoding: {})",
+                encoding
+            ));
+        scrape
+    };
+
+    let encodings = &[
```

Comment:
Add a comment that says that the case where there is no Accept-Encoding is the default and is tested in all the other tests.